### PR TITLE
[refactor] odemis gui cont streams.py file

### PR DESCRIPTION
**Pull request by nandishjpatel**
[refactor] create a streams subpackage in odemis.gui.cont, all the classes earlier in odemis.gui.cont.streams.py have been split into individual python files under the odemis.gui.cont.streams subpackage.

The refactoring will result in a better file structure.

NOTE: https://github.com/delmic/odemis/pull/2629 needs to be merged first before reviewing this PR

### CODE REVIEW

**Review by tepals — APPROVED** [review]

File: src/odemis/gui/cont/streams/stream_bar.py
Comment (pieleric) [inline src/odemis/gui/cont/streams/stream_bar.py:1]: Let's not break too much every class. I'd rather see all the children classes of `StreamBarController` into the same file, here. It's simpler to read the code, as methods are overridden and delegated.

**Review by pieleric — CHANGES_REQUESTED** [review]

File: src/odemis/gui/cont/streams/stream_bar.py
Comment (nandishjpatel) [reply src/odemis/gui/cont/streams/stream_bar.py:1]: The `stream_bar.py` is already 1056 lines, having all the children classes will add-up more. To read the code, I think one can split the IDE window (one window `stream_bar.py` second window children class). @pieleric what's your final say? I do it.

Comment (pieleric) [reply src/odemis/gui/cont/streams/stream_bar.py:1]: Please keep all the related classes in a single file.

File: src/odemis/gui/cont/streams/__init__.py
Comment (pieleric) [inline src/odemis/gui/cont/streams/__init__.py:1]: No need to create a submodule `streams` . Just place the `stream.py` and `stream_bar` in `odemis.cont`. 
This way, you can write in the tab controllers code:
`from odemis.gui.cont.stream_bar import StreamBarController`

**Review by pieleric — CHANGES_REQUESTED** [review]

File: src/odemis/gui/cont/streams/__init__.py
Comment (nandishjpatel) [reply src/odemis/gui/cont/streams/__init__.py:1]: Done!

**Comment by pieleric** [comment]
@nandishjpatel , there is a conflict (easy to fix). Would you mind fixing it? Then I will merge.

**Comment by nandishjpatel** [comment]
> @nandishjpatel , there is a conflict (easy to fix). Would you mind fixing it? Then I will merge.

Thanks! resolved now.

**Pull request merged**